brave/brave-core
# feat(wallet): Portfolio Asset Segmented Controls

## Description 
- Separates `Accounts` and `Transactions` with a `Segmented Controller` on the `Asset Details` view.
- Introduces `#` hash routing (`#accounts` or `#transactions`) will be applied to the end of `route` depending on which option you click on the `Segmented Controller`. If no `hash` is present, it will default to showing `Accounts`.


Resolves <https://github.com/brave/brave-browser/issues/30981>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to the `Portfolio` page and select an asset to view
2. By default `Accounts` should be selected
3. Switch between `Accounts` and `Transactions` and make sure everything works properly
4. Observe the `#` value in the `Route`, make sure it updates depending on your selection.

https://github.com/brave/brave-core/assets/40611140/4ecb4f35-f542-4471-957f-189b04f133d5
